### PR TITLE
Log only to console

### PR DESCRIPTION
**Pull request by dleard**
- Adds modified code from https://github.com/puckel/docker-airflow to docker folder
- change all logger handlers to 'console'
- follows / adapts instructions from https://github.com/apache/airflow/blob/1e3cdddcd87be3c0f11b43efea11cdbddaff4470/docs/howto/write-logs.rst
- Instructions under 'Writing logs to Azure Blob Storage'

### CODE REVIEW

File: cas-airflow/values.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@ airflow:
   airflow:
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.1.0'
     extraVolumeMounts:
       - name: cas-airflow-home
-        mountPath: /usr/local/airflow
+        mountPath: /usr/local/airflow/dags
```
Comment (wenzowski) [inline cas-airflow/values.yaml:8]: 👍

File: cas-airflow/values.yaml [hunk 2]
```diff
@@ -2,10 +2,10 @@ airflow:
   airflow:
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.1.0'
```
Comment (wenzowski) [inline cas-airflow/values.yaml]: while this does fit the [grammar of a semver tag](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions) maybe we could use build identifiers for pre-releases? See [spec item 10](https://semver.org/#spec-item-10) of the spec.

File: airflow/Chart.lock [hunk 3]
```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: file://../postgresql
-  version: 0.13.1-p3
+  version: 0.13.1-p4
```
Comment (wenzowski) [inline airflow/Chart.lock:4]: semver thoughts would apply here too

this is becoming more important in the context of release management strategy for our team

File: docker/cas-airflow/Dockerfile [hunk 4]
```diff
@@ -5,8 +5,21 @@
...
 USER root
+RUN set -ex \
+    && apt-get update -yqq \
+    && apt-get upgrade -yqq \
+    && apt-get install python-lxml -yqq \
+    && pip install lxml
+
+COPY root/usr/local/airflow/airflow.cfg ${AIRFLOW_USER_HOME}/airflow.cfg
+COPY root/usr/local/airflow/config/__init__.py ${AIRFLOW_USER_HOME}/__init__.py
+COPY root/usr/local/airflow/config ${AIRFLOW_USER_HOME}/config
```
Comment (wenzowski) [inline docker/cas-airflow/Dockerfile]: looks great!

File: postgresql/values.yaml [hunk 5]
```diff
@@ -64,7 +64,7 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  storageClass: netapp-block-standard
+  storageClass: netapp-file-standard
```
Comment (wenzowski) [inline postgresql/values.yaml:67]: let's try changing this back to `netapp-block-standard` and see if it's still blowing up; we really should use block storage for db if we can get it to provision properly

File: docker/cas-airflow/root/usr/local/airflow/config/logging_config.py [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    'formatters': {
+        'airflow.task': {
+            'format': LOG_FORMAT,
+        },
+    },
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'airflow.task',
+            'stream': 'ext://sys.stdout'
```
Comment (wenzowski) [inline docker/cas-airflow/root/usr/local/airflow/config/logging_config.py:53]: nice 👍

**Review by wenzowski — CHANGES_REQUESTED** [review]
notes about semver might be worth a team discussion; only minor things otherwise :)

File: postgresql/values.yaml (hunk 5 above)
Comment (dleard) [reply postgresql/values.yaml:67]: still exploding

Comment (dleard) [reply postgresql/values.yaml:67]: postgrestimeout

**Comment by dleard** [comment]
The changes to values.yaml in cas-airflow appears to copy everything to $Home/dags, so that we have all contents of $Home in the dags folder, and then the dags are in $Home/dags/dags

**Comment by wenzowski** [comment]
yes that sounds expected; we'll need to update the volume itself to move the dags up a folder level (and remove the home folder files from the volume)

**Comment by dleard** [comment]
It appears sending tasks directly to the console handler doesn't work. It needs some extra processing. Attempting to use the elasticsearch handler to stream to stdout. https://github.com/GoogleCloudPlatform/airflow-operator/issues/72

**Pull request closed without merging**